app/Presenters/Admin/ArtworkPresenter.php

```diff
@@ -453,13 +454,15 @@ protected function getArtworkAccordionBlocks()
             ];
         }
 
-        if ($this->entity->multimediaResources && $this->entity->multimediaResources->isNotEmpty()) {
-            $resultsByType = $this->entity->multimediaResources->groupBy('api_model')->sortKeys();
+        $multimediaResources = $this->entity->multimediaResources;
+        if ($multimediaResources && $multimediaResources->isNotEmpty()) {
+            $resultsByType = $multimediaResources->groupBy('api_model')->sortKeys();
             $content[] = $this->buildMultimediaBlocks($resultsByType, 'Multimedia');
         }
 
-        if ($this->entity->educationalResources && $this->entity->educationalResources->isNotEmpty()) {
-            $resultsByType = $this->entity->educationalResources->groupBy('api_model')->sort();
+        $educationalResources = $this->entity->educationalResources;
+        if ($educationalResources && $educationalResources->isNotEmpty()) {
+            $resultsByType = $educationalResources->groupBy('api_model')->sort();
```

Comment:
These two refactors above prevent triplicate api calls. The `$this->entity->multimediaResources` and `$this->entity->educationalResources` properties make an api request, so calling them three times made three separate calls. This makes sure each resource is only requested once.